Gemfile

```diff
@@ -107,7 +107,6 @@ gem "twitter", "~> 7.0" # A Ruby interface to the Twitter API
 gem "uglifier", "~> 4.2" # Uglifier minifies JavaScript files
 gem "validate_url", "~> 1.0" # Library for validating urls in Rails
 gem "vault", "~> 0.16" # Used to store secrets
-gem "warning", "~> 1.2" # Adds custom processing for warnings, including the ability to ignore specific warning messages
```

Comment:
pushed to this branch because the warning gem wasn't removed from the lockfile (and buildkite was failing).